### ISSUE TITLE
Training model

### ISSUE THREAD

**Issue by slayerduck** [nagadomi/waifu2x#4]
Do i need to use the train.lua or is convert_data.lua enough? I used convert_data.lua without any errors and trained it with 500 images but when i try the train.lua it errors out. Examples:

th train.lua -method noise -noise_level 1 -test images/miku_noise.png
{
  core : 2
  batch_size : 2
  method : "noise"
  validation_crops : 40
  noise_level : 1
  epoch : 200
  data_dir : "./data"
  test : "images/miku_noise.png"
  model_file : "./models/noise1_model.t7"
  scale : 2
  image_list : "./data/image_list.txt"
  model_dir : "./models"
  crop_size : 128
  learning_rate : 0.00025
  block_offset : 7
  images : "./data/images.t7"
  seed : 11
  validation_ratio : 0.1
}
/root/torch/install/bin/luajit: /root/waifu2x-master/lib/image_loader.lua:48: attempt to index local 'fp' (a nil value)
stack traceback:
        /root/waifu2x-master/lib/image_loader.lua:48: in function 'load_float'
        train.lua:75: in function 'train'
        train.lua:139: in main chunk
        [C]: in function 'dofile'
        /root/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:131: in main chunk
        [C]: at 0x004064d0

th cleanup_model.lua -model models/noise1_model.t7 -oformat ascii
/root/torch/install/bin/luajit: /root/torch/install/share/lua/5.1/torch/File.lua:277: unknown object
stack traceback:
        [C]: in function 'error'
        /root/torch/install/share/lua/5.1/torch/File.lua:277: in function 'readObject'
        /root/torch/install/share/lua/5.1/torch/File.lua:294: in function 'load'
        cleanup_model.lua:63: in main chunk
        [C]: in function 'dofile'
        /root/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:131: in main chunk
        [C]: at 0x004064d0

**Comment by nagadomi** [comment]
Sorry. It was my mistake.
Please try
```
th train.lua -method noise -noise_level 1 -test images/miku_noisy.png
```
`images/miku_noise.png` does not exist. `images/miku_noisy.png` is true.
